lib/trino-parquet/src/main/java/io/trino/parquet/ParquetTypeUtils.java

```diff
@@ -113,7 +113,9 @@ public static Map<List<String>, ColumnDescriptor> getDescriptors(MessageType fil
                 .stream()
                 .collect(toImmutableMap(
                         columnIO -> Arrays.asList(columnIO.getFieldPath()),
-                        PrimitiveColumnIO::getColumnDescriptor));
+                        PrimitiveColumnIO::getColumnDescriptor,
+                        // Same column name may occur more than once when the file is written by case-sensitive tools
+                        (oldValue, _) -> oldValue));
```

Comment:
This feels like a workaround with unforeseeable results.

If i have two columns "colA" string and "COLa" int in the Parquet file, I'll be probably selecting always "colA" simply because it comes first, even though the user may want actually to get an integer  from the file.

I also feel that the lowercasing of the field names in getting the file schema may need to be reconsidered.

https://github.com/trinodb/trino/blob/e09938378ba3c1b9a60f25a08b8786106bc56ade/lib/trino-parquet/src/main/java/io/trino/parquet/reader/MetadataReader.java#L265

Maybe the bigger issue we're dealing with here is https://github.com/trinodb/trino/issues/17